internal/controllers/common/finalizer_utils.go

```diff
@@ -41,8 +41,11 @@ import (
 
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+var log = logf.Log.WithName("cryostat_finalizer")
```

Comment:
Maybe this should just be called "common" since this log variable is exported to the rest of the package.

Reply:
Oh yess sure! I will change that!